File: app/src/main/java/com/example/olesya/boardgames/interfaces/ScreenCallback.java
package com.example.olesya.boardgames.interfaces;

public interface ScreenCallback {

    void onStartNewRound();

    void onShuffleEnd();

    void stopRound();

    void onAddUserEvent(String clientName);

    void onRemoveUserEvent(String clientName);

    void initHand(String username);
}

File: app/src/main/java/com/example/olesya/boardgames/interfaces/ScreenCallback.kt
package com.example.olesya.boardgames.interfaces

import com.example.olesya.boardgames.entity.Player


interface ScreenCallback {

    fun onStartNewRound()

    fun stopRound()

    fun onShuffleEnd()

    fun onAddUserEvent(username: String, renamedFrom: String): Player

    fun onRemoveUserEvent(username: String)

    fun initHand(username: String)
}